templates/vhost/_security.epp

```diff
@@ -1,6 +1,8 @@
 <IfModule mod_security2.c>
 <% if $modsec_disable_vhost {-%>
   SecRuleEngine Off
+<% } else { -%>
+  SecRuleEngine On
 <% } -%>
```

Comment:
I'm not a fan of double negatives, but I think this is better than renaming it now.
```suggestion
  SecRuleEngine <%= apache::bool2httpd(!$modsec_disable_vhost) %>
```

Reply:
Should i embark this change in my commit or using the github suggestion commit is enought ?

Reply:
I think in this repo there's no need to fix up your commits.